src/resources/postcss/components/_search.pcss

```diff
@@ -68,26 +69,82 @@
 			}
 
 			&:first-child {
-				flex: 1 1 34.5%;
+				flex: 1 1 100%;
 			}
 
 			&:nth-child(2) {
-				flex: 1 1 37.5%;
+				flex: 1 1 100%;
 			}
 
-			&:nth-child(3) {
-				flex: 1 1 28%;
+			&:last-child {
+				.tribe-common-c-search__input {
+					@media (--viewport-medium) {
+						border-right: 0;
+					}
+				}
+
 			}
+
 		}
 	}
 
 	.tribe-common-c-search__input {
+
 		@media (--viewport-medium) {
-			border: none;
 			border-right: 1px solid var(--color-border-default);
 			height: auto;
 			padding: 7px 18px 8px;
 		}
+
+		&:focus {
+			outline: none;
+		}
+	}
+
+	.tribe-common-c-search__input--keyword {
```

Comment:
we should probably put the `--keyword` modifier around the `<div>` wrapper of the label and input.

Reply:
The idea of this modifier is to actually make the input styles different, and not the label or anything else. Why do you think we should move it to the wrapper?

Reply:
The same could be said about the list items and the links in the select/tabs for the view selector. If we want to allow this to be extendable, then we should stay consistent in our approach.

Reply:
I feel this case is different, because these are inputs, and the styles here are purely related to the input. The wrapper is really generic on this one. The list items of navigations are purely related to the action they have inside.

Reply:
But I get you, with the shoutout for consistency, I agree we should make the effort to be as consistent as we can.

Reply:
@bordoni want to weigh in here? i know you had a preference about the `<li>` case last time.

Reply:
I do like it to be consistent with what we did on the other PRs where we use the class on the wrapper element.

Reply:
So, this would end up being like this: 

➡️  having a the wrapper div with the additional class, for example for the keyword: `tribe-common-form-control-text__keyword` 

- Applying the focus modifier class on the wrapper and leaving the kind of BEM format we've been following, because we'll have to target the input inside the wrapper. E.g.:

`
.tribe-common-form-control-text__keyword .tribe-common-c-search__input { /* whatever */ }
.tribe-common-form-control-text__keyword--focus .tribe-common-c-search__input { /* whatever focus state */ }
`

At some point I feel it's not consistent with the rest, independently from what we have done on the navigation, where the `<li>` was the "owner" of the action inside (a link).

Reply:
The `tribe-common-form-control-text` classes are common classes, let's not add modifiers to these as these should not know about whether there's a keyword modifier or location modifier. Instead, let's add a `tribe-common-c-search__input-control` or `__input-wrapper` class to the div that has the `tribe-common-form-control-text` class. That way, we're working within the search component context and can add `tribe-common-c-search__input-control--keyword` class.